### DISCUSSION TITLE
Issue with Paperscroller

### DISCUSSION THREAD

**Discussion by VADDIJASWANT** [clientIO/joint#2456] [Q&A]
### Introduction

Hi Guys,

I hope this message finds you well. I've encountered a challenge with the Paperscroller in my mobile views. While I can successfully scroll left and right, I'm facing issues scrolling up and down.

Here's how I've initialized the Paperscroller:
``let paperScroller = new ui.PaperScroller({
    paper: paper,
    scrollWhileDragging: true,
    contentOptions: {
        allowNewOrigin: "any",
        padding: 0
    },
    padding: 0
}); ``
I initiate panning when the pointer is down in a blank area. However, it seems there's a mistake somewhere, as vertical scrolling is not functioning as expected.

I'd greatly appreciate any insights or guidance on where I might be going wrong and how to resolve this issue.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by VADDIJASWANT** [comment]
Hi @kumilingus , I apologize for reaching you out directly here, but I wanted to bring your attention to this discussion as I initiated this about three months ago. I was stuck with some other work items at that time I really forgot about this discussion .I thought it might be helpful to tag you in hopes of gaining some insights on this issue. If you could spare some time to assist, I would greatly appreciate it.

**Reply by kumilingus** [reply]
Please provide an example, steps to reproduce or link to an existing example that suffers from the same problem. Thank you.

**Reply by VADDIJASWANT** [reply]
[JointJSPaperScroller.zip](https://github.com/clientIO/joint/files/14599855/JointJSPaperScroller.zip)
Hi @kumilingus ,
You can download this zip file and run it locally to check the issue. Please add joint-plus.min.js and joint-plus.min.css to the root folder. I didn’t include these files as they are licensed. Once you have loaded the project and opened it on mobile, you will notice that the vertical scroll is not working. If you need any additional information, feel free to ask me.
